<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DLMotionBlur</class>
 <widget class="QDialog" name="DLMotionBlur">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DLMotionBlur</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Model name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="modelNameInput">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="placeholderText">
    <string>ex: 3Depth_K201</string>
   </property>
  </widget>
  <widget class="QPushButton" name="trainButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>241</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Train model</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="renderButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>140</y>
     <width>241</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Render frame</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Convolution size</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="outputField">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>511</width>
     <height>151</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg 2';&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="placeholderText">
    <string>Python application output</string>
   </property>
  </widget>
  <widget class="QPushButton" name="settingsButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>40</y>
     <width>241</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="trainFromCheckpointCheckbox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>191</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Train from checkpoint</string>
   </property>
  </widget>
  <widget class="QPushButton" name="showGraphButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Show training graph</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="KSizeSpinBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
   <property name="value">
    <number>201</number>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>trainButton</zorder>
  <zorder>renderButton</zorder>
  <zorder>label_4</zorder>
  <zorder>outputField</zorder>
  <zorder>modelNameInput</zorder>
  <zorder>settingsButton</zorder>
  <zorder>trainFromCheckpointCheckbox</zorder>
  <zorder>showGraphButton</zorder>
  <zorder>KSizeSpinBox</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
